### PR TITLE
Bug : Sending email always clears the attachments

### PR DESCRIPTION
**Pull request by mithunbharadwaj**

### CODE REVIEW

File: asabiris/orchestration/sendemail.py [hunk 1]
```diff
@@ -52,13 +52,15 @@ async def send_email(
 		:param attachments: a list of tuples, each tuple containing the filename and the file contents
 		"""
 
+		render_failed = False
```
Comment (ateska) [inline asabiris/orchestration/sendemail.py:55]: Raise the exception, this is not a good approach.

File: asabiris/orchestration/sendemail.py [hunk 2]
```diff
@@ -52,13 +52,15 @@ async def send_email(
...
+
 		# Render a body
-		body_html, email_subject_body = await self.render(body_template, body_params, email_to)
+		body_html, email_subject_body, render_failed = await self.render(body_template, body_params, email_to, render_failed)
 
 		if email_subject is None or email_subject == '':
 			email_subject = email_subject_body
 
-		if asab.Config.get("jinja", "failsafe"):
+		if render_failed("jinja", "failsafe"):
```
Comment (ateska) [inline asabiris/orchestration/sendemail.py:63]: Is it a boolean or a function?

**Review by ateska — CHANGES_REQUESTED** [review]

File: asabiris/orchestration/sendemail.py (hunk 1 above)
Comment (mithunbharadwaj) [reply asabiris/orchestration/sendemail.py:55]: @ateska I will refactor this spaghetti code.
It's in another MR

**Comment by mithunbharadwaj** [comment]
Closing this because it will be fixed in https://github.com/TeskaLabs/asab-iris/pull/38

**Pull request closed without merging**